experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -70,7 +80,7 @@ export type LoggerProviderConfig = {
   /**
    * Reference to the LoggerRecordProcessor instance by the NodeSDK
    */
-  logRecordProcessor: LogRecordProcessor;
```

Comment:
This is an exported type, so it would be a breaking change to drop this field. Instead, as was done with `spanProcessor` in https://github.com/open-telemetry/opentelemetry-js/pull/4454, how about keeping it, but marking it deprecated. See https://github.com/open-telemetry/opentelemetry-js/pull/4454/files#diff-cf9f37462550254b82df2157b0ddaf0ba4c0a12782d46b803a076117740aac46

Reply:
I did add deprecated to SDK [config ](https://github.com/open-telemetry/opentelemetry-js/pull/4740/files#diff-cf9f37462550254b82df2157b0ddaf0ba4c0a12782d46b803a076117740aac46R35). This is exported but seems like a very internal type. I would imagine that most people have their own types for this anyways. This is experimental package so maybe we can just add it and not introduce another deprecated field that someone needs to remove later?

Reply:
> This is exported but seems like a very internal type. I would imagine that most people have their own types for this anyways.

I think you are probably right.

> This is experimental package so ...

While this is experimental, doing a breaking change (and taking the, in this case, minor version bump) is basically free from consequences.   So I don't see the harm in:
1. keeping this removal of `LoggerProviderConfig.logRecordProcessor` in your patch; and
2. marking this PR as breaking (by changing the title to `feat(sdk-node)!: ...` and moving the changelog entry to the breaking section)

It is a bit of a microscopically breaking change, but why break the rule when there is no cost. The new feature is going to result in a minor version bump anyway.

Thoughts?

Reply:
I agree. Done.